fs-storage/Cargo.toml

```diff
@@ -17,7 +17,7 @@ serde_json = "1.0.82"
 serde = { version = "1.0.138", features = ["derive"] }
 jni = { version = "0.21.1", optional = true }
 jnix = { version = "0.5.1", features = ["derive"], optional = true }
-
+bincode = "1.3"
```

Comment:
I'm not sure why do we need `bincode`. Will users be able to decode the value files themselves using simple UNIX tools? If `bincode` creates some optimized layout, then users would be required to use it for deserializing, too.

We should give users choice how they want to serialize their values. For text values though, UTF-8 is kinda standard encoding. Let's just bound the value types with `Serialize` trait.

Test code can use `bincode` or whatever is easier, probably just UTF-8 encoding for String values.